src/OrchardCore.Modules/OrchardCore.Settings/Views/Settings.Edit.cshtml

```diff
@@ -97,4 +100,19 @@
             </div>
         </div>
     </div>
+    <div class="tab-pane fade" id="cache" role="tabpanel" aria-labelledby="cache-tab">
+        <h5>@T["Settings for dynamic caches"]</h5>
+        <div class="row form-group">
+            <div class="col-xl-6" asp-validation-class-for="CacheMode">
+                <label asp-for="CacheMode">@T["Cache Mode"]</label>
+                <select asp-for="CacheMode" class="form-control">
+                    <option value="@CacheMode.FromConfiguration.ToString()">@T["From environment — enabled in <em>Production</em>, disabled otherwise"]</option>
```

Comment:
Maybe `FromEnvironment`

Reply:
I think I agree @jtkech but copied from the `Resources` where it is `FromConfiguration`, so didn't want to change it

i.e.
```
                case ResourceDebugMode.FromConfiguration:
                    options.DebugMode = !_env.IsProduction();
                    break;
```

The only change I made was to call it `CacheMode` not `CacheDebugMode` because there it was more about enabling / disabling the cache (which isn't a debug exactly, or putting it into debug).

so ended up with 
```
        FromConfiguration,
        Enabled,
        DebugEnabled,
        Disabled
```

Reply:
Okay cool then